src/exo/API_scheduling.py

```diff
@@ -715,6 +715,39 @@ def reorder_stmts(proc, block_cursor):
     loopir  = Schedules.DoReorderStmt(loopir, s1, s2).result()
     return Procedure(loopir, _provenance_eq_Procedure=proc)
 
+@sched_op([ExprCursorA(many=True)])
```

Comment:
Should this operation apply to a single location or many locations?

Reply:
I'm fine with both, do you have a suggestion?